modules/basic/ds/hashmap.h

```diff
@@ -32,6 +32,9 @@ limitations under the License.
 #include "common/util/arrow.h"
 #include "common/util/uuid.h"
 
+#include "graph/fragment/property_graph_utils.h"
```

Comment:
Why you need `#include "graph/fragment/property_graph_utils.h"` in hashmap.h? as vineyard_graph depends on vineyard_basic.